WooCommerce/Classes/ViewRelated/Orders/Order Creation/PaymentSection/ShippingLineDetails.swift

```diff
@@ -102,6 +96,38 @@ struct ShippingLineDetails: View {
     }
 }
 
+/// Adds a currency symbol to the left or right of the provided content
+///
+private struct CurrencySymbol: ViewModifier {
```

Comment:
Do you think it deserves to be public? Not necessary in this PR, when adding to an additional screen.

Reply:
Yep, that makes sense. I defaulted to private here (especially since it was a bit experimental at first) but for reuse it could be public and we could move this perhaps to `WooCommerce/Classes/View Modifiers`.